pkgs/misc/base16-builder/node-packages.nix

```diff
@@ -12,7 +12,7 @@ let
   nodeEnv = import ../../development/node-packages/node-env.nix {
     inherit (pkgs) stdenv lib python2 runCommand writeTextFile writeShellScript;
     inherit pkgs nodejs;
-    libtool = if pkgs.stdenv.isDarwin then pkgs.darwin.cctools else null;
+    libtool = if pkgs.stdenv.isDarwin then pkgs.cctools or pkgs.darwin.cctools else null;
```

Comment:
Do we really need the or here?

Reply:
good spot -- pkgs.darwin.cctools is now an alias so these should all be removed, i think. there are a number of them.

Reply:
Also don't we trigger the alias everytime unless people have aliases turned off locally?

Reply:
> I think you need to apply the patch to node2nix inside nodePackages otherwise we quickly run into problems here.

I can patch the in-tree copy once I have submitted it upstream. I don’t want this PR to cause any rebuilds, so that would be done in a separate PR.

Reply:
> Also don't we trigger the alias everytime unless people have aliases turned off locally?

Aren’t they disabled in the ofborg checks, meaning eval will fail unless the aliases aren’t used?

Reply:
Yeah, ofborg won't have them.

Derp, i forgot that package aliases don't trigger warnings, only options do.